client/src/components/Overview/ProductInformation.jsx

```diff
@@ -2,12 +2,16 @@ import React, {useState, useEffect} from 'react';
 
 const ProductInformation = () => {
 
+  const scrollToReviews = () => {
+    document.getElementsByClassName('reviews').scrollIntoView();
```

Comment:
Be sure to coordinate with Demi to make sure her component has the className: "review"

Reply:
Yep, the above line of code actually didn't work and I had to fix a few commits later.
Had to do it to index 0 since class can have more than element in comparison to doing it to ID.